pandera/typing/geopandas.py

```diff
@@ -0,0 +1,18 @@
+"""Pandera type annotations for GeoPandas."""
+from pandera.engines.pandas_engine import Geometry
+from pandera.typing.pandas import DataFrame, Series, T
+
+try:
+    import geopandas as gpd
+
+    GEOPANDAS_INSTALLED = True
+except ImportError:
+    GEOPANDAS_INSTALLED = False
+
+if GEOPANDAS_INSTALLED:
+    # pylint:disable=too-few-public-methods
+    class GeoSeries(Series[Geometry], gpd.GeoSeries):
```

Comment:
I think these need to inherit from the `typing.common` base classes and inherit from `Generic` directly, similar to how `typing.pandas.DataFrame/Series` implements it:

```python
from typing import Generic

from pandera.typing.common import DataFrameBase, SeriesBase
...

    class GeoSeries(SeriesBase, gpd.GeoSeries, Generic[Geometry]):
    
    class GeoDataFrame(DataFrameBase, gpd.GeoDataFrame, Generic[T])
```

Reply:
Ah ok fixed!